server/services/parcelService.js

```diff
@@ -2,6 +2,10 @@ const parcels = require('../../data/parcels.json')
 
 module.exports = {
   get: function () {
-    return parcels.map(parcel => ({ ref: parcel.parcelRef }))
+    return parcels
```

Comment:
This breaks the front-end which is expecting /parcels to return { parcels: [{ref:string}] } as per https://eaflood.atlassian.net/browse/FCEP-47

Reply:
Good point. I thought it was a mistake that the transformation hadn't been removed when the parcel data was expanded to include more than just `ref`. I guess we should update the parcel data to use `ref`, rather than `parcelRef` so that everything is consistent?

Reply:
Rather than go back to pulling out and renaming one property of each parcel here, I've updated `parcel.json` to name the property just `ref` instead of `parcelRef`. Should mean everywhere can just use `parcel.ref` now.

Reply:
Yes, that should be the path of least resistance, reverting to the old code will break the updated tests.